lib/js/components/SelectList/SelectListItemTile/SelectListItemTile.stories.ts

```diff
@@ -0,0 +1,57 @@
+import SelectListItemTile from './SelectListItemTile.vue';
+import { TILE_STATES } from './../../Tile/Tile.consts';
+
+import { Args, Meta, StoryFn } from '@storybook/vue3';
+import { args, argTypes, data, template } from './../../Tile/Tile.sb.shared';
+
+export default {
+	title: 'Components/SelectListItemTile',
```

Comment:
why not in `SelectList` like others?
```suggestion
	title: 'Components/SelectList/SelectListItemTile',
```